.travis.yml

```diff
@@ -0,0 +1,24 @@
+# @Date:   2017-03-20T16:36:18-04:00
+# @Project: scatt-jacc-caj
+# @Filename: travis.yml
+# @Last modified time: 2017-03-20T16:53:05-04:00
+
+#defining the language used
+language: java
+
+script:
+#installing dependencies
+- ./gradlew assemble
+#calling default script
+- ./gradlew check

```

Comment:
My understanding is that `gradlew build` performs all steps (fetching dependencies through running tests). Am I mistaken, or is there another reason that you didn't use that as your verification step?